### PR TITLE
Adds link to the wiki on deployment with Ansible

### PR DESCRIPTION
**Pull request by filipefurtad0**
Adds link to the wiki on deployment with Ansiable

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,6 +65,9 @@ Some playbooks require third-party roles, which are specified in `bin/requiremen
 $ bin/setup
 ```
 
+## Deployment with Ansiable
```
Comment (sigmundpetersen) [inline README.md:68]:
```suggestion
## Deployment with Ansible
```

**Comment by dacook** [comment]
Thanks for working to make sure this is up to date @filipefurtad0 . What do you think of my suggested change in the new commit?

File: README.md (hunk 1 above)
Comment (filipefurtad0) [reply README.md]: I did not spot that one :sweat_smile:  I've taken the name from [here](https://github.com/openfoodfoundation/ofn-install/wiki/Deployment-with-Ansiable) -> I've fixed it it now on the wiki, as well.

**Comment by filipefurtad0** [comment]
> Thanks for working to make sure this is up to date @filipefurtad0 . What do you think of my suggested change in the new commit?

Sure @dacook - I agree, let's rather go with that option :+1:

File: README.md (hunk 1 above)
Comment (sigmundpetersen) [reply README.md]: I think it should be 'Ansible' (https://www.ansible.com/)

Comment (filipefurtad0) [reply README.md]: > I think it should be 'Ansible' (https://www.ansible.com/)

Of course... Sorry, for my mix up.

**Pull request merged**